src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -171,14 +171,6 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
             }
         });
 
-        add(new GeneralAction(Actions.MERGE_DOI, Localization.lang("Get BibTeX data from DOI")) {
```

Comment:
I'm not really seeing the problem. A button somewhere else won't help the (superior) habit of right-clicking on an entry and selecting in a menu compared to double-clicking to open the editor, change to the right tab and clicking a button. Marking, click on another button is slightly better, but I still do not see what the problem with the right-click menu staying is. "New from DOI" is not going to end up in the right-click menu? 

I thought the purpose of #1098 was to add e new entry (which IMHO would be better splitting web search and "add/fetch") so I do not see marking and clicking that button as a replacement. 

The context here is that one do get slightly different information when searching and when adding information from DOI, so I usually do both at once. Not to mention improving all my old bib-files.